OpenUpSA/wazimap-ng
# Send email only for failed tasks

## Description
Send email only for failed tasks

## Related Issue
#238

## How to test it locally
do a successful upload and check in qcluster that no email will be sent.
Now add variable in datasets/dataloader that in undefined so task will throw error and check q cluster for email being sent for task failure.

## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
